### PR TITLE
adding fix to check if ssl i valid

### PR DESCRIPTION
**Pull request by iruzevic**
problem occurs if you are on https but don't have valid ssl certificate. 
wp_remote_get can't get file and cant show assets and we don't want that

### CODE REVIEW

File: wp-content/themes/inf_theme/functions.php [hunk 1]
```diff
@@ -31,7 +31,7 @@
  * @since 3.0.0
  * @package Inf_Theme
  */
-define( 'INF_ASSETS_PUBLIC_URL', get_template_directory_uri() . '/skin/public/' );
+define( 'INF_ASSETS_PUBLIC_PATH', get_template_directory() . '/skin/public/' );
```
Comment (dingo-d) [inline wp-content/themes/inf_theme/functions.php]: The `INF_ASSETS_PUBLIC_PATH` should be in the rename script as well, no?

Comment (iruzevic) [reply wp-content/themes/inf_theme/functions.php]: it should, could you please add it?

Comment (dingo-d) [reply wp-content/themes/inf_theme/functions.php]: Will do :+1:

**Comment by dingo-d** [comment]
This is good to go? If it is, just merge it :)

**Pull request merged**